project/StreamOperatorsIndexGenerator.scala

```diff
@@ -261,7 +263,7 @@ object StreamOperatorsIndexGenerator extends AutoPlugin {
       (description, categoryName)
     } catch {
       case NonFatal(ex) =>
-        throw new RuntimeException(s"Unable to extract details from $file", ex)
+        throw new RuntimeException(s"Unable to extract details from $file, error:${ex.getMessage}", ex)
```

Comment:
Missing a space after colon, but why add the message to the wrapper message at all? It's provided as the cause already.

Reply:
It's not showing up in the console.